packages/app/hooks/useChannelContext.ts

```diff
@@ -156,7 +156,13 @@ export const useChannelContext = ({
     if (!channelQuery.data) {
       return;
     }
-    navigate('/search/' + channelQuery.data.id);
+    if (isGroupChannelId(channelQuery.data.id)) {
```

Comment:
Can we break out anything shared between this and `useChannelContext.native.ts`? Maybe there's just one `useChannelContext` and a split `useChannelNavigation`?